lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1641,9 +1546,7 @@ sub _failure_reason {
             # Look for serial failures which have bug reference
             my @bugrefs = map { find_bugref($_->{title}) || '' } @{$m->details->{results}};
             # If bug reference is in title, put it as a failure reason, otherwise use module name
-            if (my $failure_reason = join('', @bugrefs)) {
-                return $failure_reason;
-            }
+            if (my $failure_reason = join('', @bugrefs)) { return $failure_reason }
```

Comment:
I'd prefer `return join('', @bugrefs) if @bugrefs;`.

Reply:
yes, that's good. Thank you.